src/main/java/com/example/lesson6/zad4/DaysOfTheWeek.java

```diff
@@ -0,0 +1,17 @@
+package com.example.lesson6.zad4;
+
+public class DaysOfTheWeek {
+
+    public static String getDaysOfTheWeek(int numberDay){
+        String nameDay;
+        if (numberDay < 6){
```

Comment:
Brakuje Ci warunku dolnego, jak podam tej metodzie 0 albo nawet -30, to zwróci mi Monday. 
Tak właściwie to w zadaniu masz zwrócić odpowiedni dzień tygodnia np. Monday, czyli powinieneś zwrócić 1 -> Monday
2 -> Tuesday, zastanów się czy w takim razie Ify są odpowiednie, może lepiej użyć konstrukcji switch :) Reszty nie sprawdzam do czasu poprawy logiki :)